tests/integration/ceph_mgr_test.go

```diff
@@ -205,6 +206,31 @@ func (s *CephMgrSuite) waitForOrchestrationModule() {
 	}
 	require.Nil(s.T(), err)
 }
+
+func (s *CephMgrSuite) CompareNodes(orchestratorNodes []string) error {
+	// get the k8s nodes
+	nodes, err := k8sutil.GetNodeHostNames(s.k8sh.Clientset)
+	if err == nil {
```

Comment:
If the error is returned immediately, a deeper code block is not needed
```suggestion
	if err != nil {
		return err
	}
```

Reply:
ok I understand .. in that way we reduce indentation levels.. probably more readable. ( noted for next times) thx!